apps/wallet/src/engine/runners/solana/collectibles.ts

```diff
@@ -159,21 +158,27 @@ export const updateRelatedCollection = async (
 		collectible.collectionId,
 	);
 
-	if (!storedCollection) {
-		const collectionMetadata: GenericNft = await mpl.nfts().findByMint({
+	if (
+		!storedCollection ||
+		!storedCollection.metadata ||
+		!('name' in storedCollection.metadata)
```

Comment:
what is the reason behind this condition: field `name` not in `storedCollection.metadata`

Reply:
there is a case that `storedCollection` is an empty `{}` and as you know, `!{}` is `false` so it will pass the `!storedCollection.metadata` check